### ISSUE TITLE
Bonding timeout

### ISSUE THREAD

**Issue by fmikulu** [iqrfsdk/clibdpa#29]
When solving #28 we identified special timing requirement for bonding:

bonding blocks for 10s so max wait 11s if user defines timeout then use user timeout

We droped it in #28 and it is covered here

**Comment by fmikulu** [comment]
Implemented b77c81fa4a159b3ea6740ce7b2c54eb73138fe27

**Comment by fmikulu** [comment]
So we need special timeout for bonding. We have these commands for coordinator and there is more commands regarding bonding:

```cpp
#define	CMD_COORDINATOR_ADDR_INFO  0
#define	CMD_COORDINATOR_DISCOVERED_DEVICES 1
#define	CMD_COORDINATOR_BONDED_DEVICES 2
#define	CMD_COORDINATOR_CLEAR_ALL_BONDS 3
#define	CMD_COORDINATOR_BOND_NODE 4
#define	CMD_COORDINATOR_REMOVE_BOND 5
#define	CMD_COORDINATOR_REBOND_NODE 6
#define	CMD_COORDINATOR_DISCOVERY 7
#define	CMD_COORDINATOR_SET_DPAPARAMS 8
#define	CMD_COORDINATOR_SET_HOPS 9
#define	CMD_COORDINATOR_DISCOVERY_DATA 10
#define	CMD_COORDINATOR_BACKUP 11
#define	CMD_COORDINATOR_RESTORE 12
#define	CMD_COORDINATOR_AUTHORIZE_BOND 13
#define	CMD_COORDINATOR_BRIDGE 14
#define	CMD_COORDINATOR_READ_REMOTELY_BONDED_MID 15
#define	CMD_COORDINATOR_CLEAR_REMOTELY_BONDED_MID 16
#define	CMD_COORDINATOR_ENABLE_REMOTE_BONDING 17
#define	CMD_COORDINATOR_SMART_CONNECT 18
```
@spinarr  Which of them are required to get special timing 11 sec, if a user doesn't provide user timeout?

**Comment by spinarr** [comment]
#define	CMD_COORDINATOR_BOND_NODE 4

**Comment by fmikulu** [comment]
Corrected by: f5a2585c82d1f4588d9b8543babd2ea72dd01428
